src/components/DisbursementDetails/index.tsx

```diff
@@ -244,6 +247,17 @@ export const DisbursementDetails: React.FC<DisbursementDetailsProps> = ({
             </div>
           </div>
 
+          <div>
+            <label className="Label Label--sm">Future balance</label>
+            <div
+              className={`DisbursementDetailsFields__value ${
+                futureBalance >= 0 ? "" : "DisbursementDetailsFields__negative"
+              }`}
+            >
+              {futureBalance}
```

Comment:
Should we format this asset amount?  The `AssetAmount` component can do that.

Reply:
I don't think so since we are showing the asset selected above. What do you think?

Reply:
I think we should at least format the amount to match the rest of the app. We could omit the asset code. I think it's more clear which balance we're talking about there if the asset code is included, though.

Before
![image](https://github.com/stellar/stellar-disbursement-platform-frontend/assets/7346473/30c6b0da-c201-4631-a0af-684b00417dbf)

After
![image](https://github.com/stellar/stellar-disbursement-platform-frontend/assets/7346473/8760bd2d-b647-4d8c-924a-c26d3f3e5538)